src/SourceBuild/patches/runtime/0002-Revert-switch-to-self-hosted-NativeAOT-compiler.patch

```diff
@@ -0,0 +1,31 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Matt Thalman <mthalman@microsoft.com>
+Date: Mon, 20 Mar 2023 15:35:53 -0500
+Subject: [PATCH] Revert switch to self-hosted NativeAOT compiler
+
+Backport: https://github.com/dotnet/runtime/issues/83695
```

Comment:
I see this issue has another suggested workaround to unblock source-build.  Thoughts on utilizing that instead?